cmd/kubelet/app/options/options.go

```diff
@@ -39,6 +39,7 @@ import (
 const (
 	DefaultKubeletPodsDirName       = "pods"
 	DefaultKubeletVolumesDirName    = "volumes"
+	DefaultKubeletDevicesDirName    = "devices"
```

Comment:
Should this be something like "volumeDevices" to distinguish it from generic devices?

Reply:
ack.